src/combinatorics/simple_graph/basic.lean

```diff
@@ -410,12 +410,12 @@ by { ext v w, simp only [from_edge_set_adj, set.mem_empty_iff_false, false_and,
 @[simp] lemma from_edge_set_univ : from_edge_set (set.univ : set (sym2 V)) = ⊤ :=
 by { ext v w, simp only [from_edge_set_adj, set.mem_univ, true_and, top_adj] }
 
-@[simp] lemma from_edge_set_inf (s t : set (sym2 V)) :
-  from_edge_set s ⊓ from_edge_set t = from_edge_set (s ∩ t) :=
+@[simp] lemma from_edge_set_inter (s t : set (sym2 V)) :
+  from_edge_set (s ∩ t) = from_edge_set s ⊓ from_edge_set t :=
```

Comment:
I think this is the wrong direction in practice. The reason it undistributes `from_edge_set` is that I expect most of the time there is more you can say about unions of sets than infima of graphs.

Reply:
I find distributivity lemmas often more useful because they let the operations distribute further along the chain. simp undistributivity lemmas are *very* rare for a reason.

In that case, we now have a bunch of lemmas distributing infimum over other graph operations, so this simp lemma is in the way, as it stands.